.github/workflows/node-integrate.yml

```diff
@@ -496,6 +503,70 @@ jobs:
             git push --tags
           fi
 
+  integrateTestGoDevMode:
+    name: Integrate Dev Mode Extension
+    runs-on: ubuntu-latest
+    needs: [linuxNode16, linuxNode14]
+    if: needs.linuxNode16.outputs.pathChangedTestGoDevMode == 'true'
+    timeout-minutes: 5 # Default is 360
+    env:
+      AWS_REGION: us-east-1
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      SLS_ORG_ID: ${{ secrets.SLS_ORG_ID }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          # Ensure to have complete history of commits pushed with given push operation
+          # It's loose and imperfect assumption that no more than 30 commits will be pushed at once
+          fetch-depth: 30
+          # Tag needs to be pushed with real user token, otherwise pushed tag won't trigger the actions workflow
+          # Hence we're passing 'serverless-ci' user authentication token
+          token: ${{ secrets.USER_GITHUB_TOKEN }}
+
+      - name: Retrieve ~/.npm from cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-v16-${{ runner.os }}-${{ github.ref }}-${{ hashFiles('node/**package*.json') }}
+
+      - name: Retrieve node_modules from cache
+        id: cacheNodeModules
+        uses: actions/cache@v2
+        with:
+          path: node/node_modules
+          key: node-modules-v16-${{ runner.os }}-${{ github.ref }}-${{ hashFiles('node/package.json') }}
+
+      - name: Install Node.js and npm
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+
+      - name: Install node dependencies
+        run: |
+          cd node
+          npm install
```

Comment:
Let's use same install command as in case of other steps, so: `npm update --save-dev --no-save`

Reply:
Ohh my bad I didn't see the `Install main project dependencies` section listed above. This should be updated to the exact command and name 👍